lib/client/terminal/terminal_unix.go

```diff
@@ -152,6 +152,15 @@ func (t *Terminal) Stderr() io.Writer {
 	return t.stderr
 }
 
+// Clear clears the terminal, including scrollback.
+func (t *Terminal) Clear() error {
+	const resetPattern = "\x1b[3J\x1b\x63\n"
```

Comment:
Maybe leave a comment linking to the source of this escape code so we know where to look if we need to change it?

Reply:
Good plan. @gzdunek can you add this?

Reply:
I think some of it came from here: https://superuser.com/questions/122911/what-commands-can-i-use-to-reset-and-clear-my-terminal

But the above looks different, @gzdunek how did you arrive at that escape sequence?

Reply:
`/x1b[3J` - clears scrollback (https://newbedev.com/how-do-i-reset-the-scrollback-in-the-terminal-via-a-shell-command https://apple.stackexchange.com/a/187773) - it is needed at least for the Mac terminal
> If n is 3, clear entire screen and delete all lines saved in the scrollback buffer (this feature was added for xterm and is supported by other terminal applications)."


`\x1b\x63` - clears current screen - same as `'\0033\0143'` linked above

Add it as a comment?

Reply:
Added